Gir_Gtk.toml

```diff
@@ -28,6 +28,7 @@ generate = [
     "Gtk.ButtonBox",
     "Gtk.Calendar",
     "Gtk.CheckButton",
+    "Gtk.Clipboard",
```

Comment:
This really needed?

Reply:
For eyeball tests, yes :-P  I wanted to check that GtkClipboard is emitted without the class struct (it doesn't have a public one).

Reply:
At some point I'd like to refactor the various stages of gir so that small-scale tests are possible.  Currently it must run from the beginning, with "real" input files.

Reply:
Ok